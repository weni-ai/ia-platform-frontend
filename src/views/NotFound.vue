<template>
  <section class="not-found">
    <h1>Page Not Found</h1>
  </section>
</template>

<script>
export default {
  name: 'NotFound',
};
</script>

<style lang="scss" scoped>
.not-found {
  height: 100vh;
  display: flex;
  align-items: center;
  justify-content: center;
}
</style>
