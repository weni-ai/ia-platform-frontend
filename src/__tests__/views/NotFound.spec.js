import { mount } from '@vue/test-utils';
import NotFound from '@/views/NotFound.vue';

describe('NotFound.vue', () => {
  let wrapper;

  beforeEach(() => {
    wrapper = mount(NotFound);
  });

  test('renders correctly', () => {
    expect(wrapper.exists()).toBe(true);
  });

  test('contains the correct elements', () => {
    const section = wrapper.find('section.not-found');
    expect(section.exists()).toBe(true);

    const heading = wrapper.find('h1');
    expect(heading.exists()).toBe(true);
    expect(heading.text()).toBe('Page Not Found');
  });

  test('matches the snapshot', () => {
    expect(wrapper.html()).toMatchSnapshot();
  });
});
