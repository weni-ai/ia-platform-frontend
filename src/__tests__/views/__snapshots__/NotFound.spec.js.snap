// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`NotFound.vue > matches the snapshot 1`] = `
"<section data-v-58703ab4="" class="not-found">
  <h1 data-v-58703ab4="">Page Not Found</h1>
</section>"
`;
